doc_admin/AdminManual.adoc

```diff
@@ -423,80 +434,36 @@ correctly perform any future Update process (see <<update-open-hospital,Update O
-* *rsc* – contains the *resources* of Open Hospital, like configuration files and other stuff
-* *rsc/icons* – contains the *icons* used in Open Hospital for windows and buttons
-* *rsc/images* – contains the *images* used in Open Hospital
-* *sql* – contains the scripts and configurations for MySQL DB
-
-These folders may be organized differently depending on the version of the software.
-
-=== 4.1 Folder tree in Open Hospital client (multiarch)
-
-In the Open Hospital zip file the following folder tree structure is found in the downloaded archive, where *_<version>_* is replaced with the current Open Hospital version:
-
-* *OpenHospital-_<version>_-multiarch-client*/
-** *bin*
-** *bundle*
-** *doc*
-** *lib*
-** *logs* (after first startup)
-** *rpt*
-** *rsc*
-** *sql*
-** CHANGELOG - Last version brief changelog
-** LICENSE - GNU GENERAL PUBLIC LICENSE V3
-** oh.ico - icon image for launcher
-** README.md
-** SetupGSM.cmd *Utility for Windows*
-** SetupGSM.sh *Utility for Linux*
-** startup.cmd *Launcher for Windows*
-** startup.sh *Launcher for Linux*
-
-=== 4.2 Folder tree in Open Hospital portable for Linux (32 and 64bit)
-
-In Open Hospital portable for Linux the following folder tree structure is found in the downloaded archive, where *_<version>_* is replaced with the current Open Hospital version and *_<arch>_* with the OS architecture (x32 or x64):
-
-* *OpenHospital-_<version>_-linux_i686-portable*/ or *OpenHospital-_<version>_-linux_x86_64-portable*/
-** data/ - folder for DB, DICOM files and logs
-** etc/ - working folder
-** _jdk-11.0.9+11-jre/_ (64bit) or _zulu11.43.21-ca-jre11.0.9-linux_i686/_ (32bit) - Java Virtual Machine
-** _mysql-<version>-<arch>-<build>/_ or _mariadb-<version>-<arch>/_ - MySQL Server or MariaDB Server
-** *oh/ the OpenHospital subfolder structure*
-** sql/ - working folder
-** CHANGELOG - Last version brief changelog
-** LICENSE - GNU GENERAL PUBLIC LICENSE V3
-** oh.ico - icon image for launcher
-** *oh.sh Launcher for OH-Linux*
-** POH-linux-changelog.md (changelog portable-linux)
-** POH-README.md (README portable-linux)
-
-=== 4.3 Folder tree in Open Hospital portable for Windows (32bit)
-
-In Open Hospital portable for Windows the following folder tree structure is found in the downloaded archive, where *_<version>_* is replaced with the current Open Hospital version:
-
-* *OpenHospital-_<version>_-win32-portable*/
-** data/ - folder for DB, DICOM files and logs
-** _jdk-11.0.9+11-jre/_ (64bit) or _zulu11.43.21-ca-jre11.0.9-linux_i686/_ (32bit) - Java Virtual Machine
+* *oh* – contains the Open Hospital application
+* *oh/bin* – contains the Open Hospital *binaries* (compiled software)
+* *oh/rpt* – contains the JasperReports® *reports* used in Open Hospital
+* *oh/rpt/PDF* – contains the reports generated by Open Hospital in PDF format
+* *oh/rsc* – contains the *resources* of Open Hospital, like configuration files and other stuff
```

Comment:
I like it: `stuff` is so exact :) How about `and other related files`?

Reply:
:-) fixed